README.md

```diff
@@ -0,0 +1,288 @@
+                                                          КП ППвИС(часть 1)
```

Comment:
Use markdown styles for format file. Readme file should be readable
![image](https://user-images.githubusercontent.com/8680298/96879535-d6fbd900-1484-11eb-9621-cd6e7753a7df.png)